lib/nylas/v2/constraints.rb

```diff
@@ -0,0 +1,44 @@
+module Nylas
+  module V2
+    class Constraints
+      attr_accessor :where, :limit, :offset, :view, :per_page
```

Comment:
Is this the only way to define filters? How are people going to define custom filters, say for example for filtering contacts on a specific address?

Reply:
This is so you can have syntax like: `api.contacts.where(street_address: "123 N West St").limit(10).offset(25)` to get the next 10 people who are in `123 N West St`

This is part of the providing an ActiveModel/ActiveRecord/ActiveResource type syntax for users of the 4.0 SDK.